wgengine/router/ifconfig_windows.go

```diff
@@ -138,15 +139,18 @@ func monitorDefaultRoutes(device *device.Device, autoMTU bool, tun *tun.NativeTu
 			tun.ForceMTU(int(iface.NlMtu)) //TODO: it sort of breaks the model with v6 mtu and v4 mtu being different. Just set v4 one for now.
 			iface, err = winipcfg.GetIpInterface(ourLuid, winipcfg.AF_INET6)
 			if err != nil {
-				return err
-			}
-			iface.NlMtu = mtu - 80
-			if iface.NlMtu < 1280 {
-				iface.NlMtu = 1280
-			}
-			err = iface.Set()
-			if err != nil {
-				return err
+				if !isMissingIPv6Err(err) {
```

Comment:
Should do the same thing when looking up the v4 interface. Presumably a v6-only network would fail on v4.

Reply:
We have some many IPv4 assumptions all over our code base that I don't want to even pretend we work without it.

If that's a goal it'll be its own big project one day.